creator-node/src/index.ts

```diff
@@ -111,30 +148,21 @@ const startApp = async () => {
     )
   }
 
-  await connectToDBAndRunMigrations()
-
   const nodeMode = config.get('devMode') ? 'Dev Mode' : 'Production Mode'
+
   await serviceRegistry.initServices()
   logger.info(`Initialized services (Node running in ${nodeMode})`)
-
   const appInfo = initializeApp(getPort(), serviceRegistry)
   logger.info('Initialized app and server')
 
-  // Clear all redis locks
-  try {
-    await redisClient.WalletWriteLock.clearWriteLocks()
-  } catch (e: any) {
-    logger.warn(`Could not clear write locks. Skipping..: ${e.message}`)
-  }
-
   // Initialize services that do not require the server, but do not need to be awaited.
   serviceRegistry.initServicesAsynchronously()
 
   // Some Services cannot start until server is up. Start them now
   // No need to await on this as this process can take a while and can run in the background
   serviceRegistry.initServicesThatRequireServer(appInfo.app)
 
-  // when app terminates, close down any open DB connections gracefully
+  // When app terminates, close down any open DB connections gracefully
   ON_DEATH((signal: any, error: any) => {
```

Comment:
does this termination correctly notify primary process?

Reply:
I'm not too sure how to test this 🤔 I verified that workers respawn properly when using `kill -9`, but that wouldn't allow for the graceful shutdown that this logic runs since it's like a force kill

Reply:
try throwing an uncaught error inside worker, or process.exit(1)?
also another thought - we'd want this inside master as well?

Reply:
tested process.exit(1) and it's working! we shouldn't need ON_DEATH in master because it doesn't have an express app to gracefully shut down. same with sequelize db connections

Reply:
if master goes down the whole thing actually stops because every other process is a child of master. so that's just like if this wasn't using cluster and the main process terminated, then the whole app stops. I just confirmed this locally

Reply:
ahh cool

Reply:
worth documenting this

Reply:
added a comment ✅